.github/workflows/rust.yml

```diff
@@ -309,21 +309,22 @@ jobs:
           toolchain: ${{ matrix.toolchain }}
 
       - name: Install tools
-        if: github.event_name == 'pull_request'
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/heads/trunk-merge/') || github.event_name == 'pull_request' }}
```

Comment:
I know that `&&` binds stronger than `||`, but would it be possible to introduce parenthesis to clarify things?

Reply:
I think that's better, yes. I also changed other location where I added this pattern previously: e5f6781da132eb5d45821099bb8043f8ce215132